httcp/config/triggers.py

```diff
@@ -42,7 +42,7 @@ def add_triggers_2017(config: od.Config) -> None:
                     min_pt=28.0,
                     # filter names:
                     # hltEle32WPTightGsfTrackIsoFilter
-                    trigger_bits=2 + 1024,
+                    trigger_bits=[2 + 1024],
```

Comment:
should it rather be 
```
trigger_bits=[2, 10]
```

Reply:
should be like [2,10] cos we are stating here bit positions.